vassal-app/src/main/java/VASSAL/tools/image/ImageUtils.java

```diff
@@ -463,4 +467,60 @@ public static String stripImageSuffix(String name) {
     }
     return name;
   }
+
+  /**
+   * Create a viewable representation of a null or empty image to
+   * use as a place holder in Configurers.
+   *
+   * The image will contain the translated text for the key
+   * Editor.ImageUtils.no_image
+   *
+   * @return Viewable null image
+   */
+  public static Image createViewableNullImage() {
```

Comment:
Would be better to return a BufferedImage, since most everything dealing with images wants one of those.

Reply:
True. Fixed.